code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -114,24 +115,44 @@
 	if(skip_contents && transfer_contents)
```

Comment:
Can you rename one of these? It's not intuitive as to what they do given the context of...
"Wait so you're going to skip the contents AND transfer the contents?"

Reply:
Yeah that's been giving me a headache lately as well, I just hadn't thought of anything better to swap it to. `skip_content_check` ought to work better.